ios/Plugin/Plugin.swift

```diff
@@ -282,9 +290,9 @@ public class DatePickerPlugin: CAPPlugin {
     
     @objc func titleChange(_ date: Date) -> String{
         if (self.pickerTitle == nil) {
-            var format: String = "E, MMM d, yyyy HH:mm"
+            var format: String = self.picker24h ? "E, MMM d, yyyy HH:mm" : "E, MMM d, yyyy hh:mm a"
             if (self.pickerMode == "time") {
-                format = "HH:mm a"
+                format = self.picker24h ? "HH:mm" : "hh:mm a"
```

Comment:
Is the 24h flag even needed? You could use the system settings via the dateFormatter

```swift
dateFormatter.dateStyle = .none
dateFormatter.timeStyle = .short
dateFormatter.string(from: date) // HH:mm or hh:mm a 
```

and maybe similar on the date behaviour

```swift
dateFormatter.dateStyle = .full
dateFormatter.timeStyle = .short
dateFormatter.string(from: date)
```
Quick example https://repl.it/@Jannnnnn/TatteredOvercookedActiveserverpages

Reply:
I liked that, we can try.